rpr.gemspec

```diff
@@ -23,4 +23,12 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler", "~> 1.11"
   spec.add_development_dependency "rake", "~> 10.0"
+
+  # testing
+  spec.add_development_dependency "minitest", "~> 5.9.0"
+  spec.add_development_dependency "minitest-power_assert", "~> 0.2.0"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)